webpack/JobWizard/steps/form/Formatter.js

```diff
@@ -56,7 +56,11 @@ export const formatter = (input, values, setValue) => {
   const value = values[name];
 
   if (isSelectType) {
-    const options = input.options.split(/\r?\n/).map(option => option.trim());
+    let { options } = input;
+    if (typeof options === 'string' || options instanceof String) {
```

Comment:
can you give an example of how the options look?
when does `options instanceof String` needed?

Reply:
Actually might change it to `if(inputType === 'plain')`
When type is plain the options need to be:
'a b c'
In the template